criu/page-xfer.c

```diff
@@ -1242,6 +1251,10 @@ int connect_to_page_server_to_recv(int epfd)
 
 int request_remote_pages(unsigned long img_id, unsigned long addr, int nr_pages)
 {
+	int flags = MSG_DONTWAIT;
+#ifdef MSG_ZEROCOPY
```

Comment:
this should be a runtime check. CRIU can't be compiled in a docker container which is running on an old kernel.